src/Core/View.js

```diff
@@ -250,9 +250,10 @@ function _preprocessLayer(view, layer, provider) {
  * @param {Layer=} parentLayer
  * @return {Promise} a promise resolved with the new layer object when it is fully initialized
  */
+
 View.prototype.addLayer = function addLayer(layer, parentLayer) {
     const duplicate = this.getLayers((l => l.id == layer.id));
-    if (duplicate.length > 0) {
+    if (duplicate.length > 0 && !parentLayer) {
```

Comment:
I think I'll close this PR, because we have to think about it.

But I think that the rendering properties {visible, color, elevation, opacity} and the data {wmts, image, wms} should be separated. Like materials and textures, the texture can be reused in several materials. WDYT?